packages/sanity/src/core/releases/__telemetry__/releases.telemetry.ts

```diff
@@ -101,3 +108,12 @@ export const UnarchivedRelease = defineEvent({
   version: 1,
   description: 'User unarchived a release',
 })
+
+/** When a release is successfully reverted
+ * @internal
+ */
+export const RevertRelease = defineEvent<RevertInfo>({
+  name: 'Revert release',
```

Comment:
Telemetry events should be Title Cased (and are usially written in past tense I believe)

From our guidelines:
> Should be in the form of <Object> <Action>, e.g. Document Published, Studio Entered, or Trial Started, and be capitalized.

```suggestion
  name: 'Release Reverted',
```

I see that we lack a bit of consistency here already, so feel free to create a ticket and leave it as a separate task

Reply:
Good call - yeh we aren't following these guidelines for the release telemetry. I've created a separate task to switch all of these to be correct past tense